README.rst

```diff
@@ -500,6 +500,21 @@ Options:
 Note: If no content is specified with ``file`` or ``files`` then
 ``rootpath`` and ``mountpoint`` are used to provide the content.
 
+fip
+***
+Generates a Firmware Image Package (FIP). A format used to bundle
+firmware to be loaded by ARM Trusted Firmware.
+
+Options:
+
+:extraargs:		Extra arguments passed to fiptool
+:fw-config:		Firmware Configuration (device tree), usually provided by BL2 (Trusted Firmware)
+:nt-fw:			Non-Trusted Firmware (BL33)
+:hw-config:		Hardware Configuration (device tree), passed to BL33
+:tos-fw:		Trusted OS (BL32) binaries. Second and third binary are used as
+			extra1 and extra2 binaries if specified. Example:
+			``tos-fw = {"tee-header_v2.bin", "tee-pager_v2.bin", "tee-pageable_v2.bin"}``
```

Comment:
Hmm, I'm not sure I like the special handling for `os-fw*` here. Why a list and not separate options?

And this selection of fiptool options seems pretty arbitrary. What about the rest? `extraargs` is not really good for anything that has a file argument.

Reply:
I selected the ones I actually need for generating valid boot images for a STM32MP15/STM32MP13. Should I just expose everything currently supported?

We could have separate `tos-fw=`, `tos-fw-extra1=`, `tos-fw-extra2=`, but I dislike this extra1/extra2 naming scheme. I like the UX of this better.

Reply:
I'd like to have all of them. Your just need to add them to the option list anyways. And copy the help text from `fiptool`.

I suppose it's ok to keep the list for `tos-fw`.

Reply:
I thought people interested in them could do it better, e.g.
```
cert = { tb-fw = "someting", scp-fw = "somethingelse" }
```
instead of
```
tb-fw-cert = "something"
scp-fw-cert = "somethingelse"
```
But I don't want to do that myself, so I will just copy them 1:1.

Reply:
Done.